etc/tendrl/alerting.conf.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I feel these are sample files and they should be named accordingly. Also they should kept under `/etc/tendrl/alerting/alerting.cond.yaml.sample` I feel. Same applies to log configurations as well.